packages/core/addon/components/table-renderer-vertical-collection.js

```diff
@@ -45,9 +40,14 @@ export default Component.extend({
   classNames: ['table-widget-container', 'table-widget-container--vc'],
 
   /**
-   * @property {Service} resize - Resize service to respond to view resize events
+   * @property {ResizeObserver} -- chrome resize observer for header changes
```

Comment:
Is this an object? `@property {Object} `??

Reply:
It's an object of the global ResizeObserver class.